.github/workflows/tests.yaml

```diff
@@ -17,8 +17,14 @@ env:
   SHOWCASE_VERSION: 0.35.0
   PROTOC_VERSION: 3.20.2
   OLDEST_PYTHON: 3.7
-  LATEST_STABLE_PYTHON: 3.13
+  LATEST_STABLE_PYTHON: 3.14
+  # TOOD(https://github.com/googleapis/gapic-generator-python/issues/2450):
```

Comment:
```suggestion
  # TODO (https://github.com/googleapis/gapic-generator-python/issues/2450):
```

Reply:
Fixed in https://github.com/googleapis/gapic-generator-python/pull/2451/commits/6dbb45b8d5c7f1c35f5313f55e9e41935ead0216